File: crm-mobile/shared/src/commonMain/kotlin/com/gurucrm/mobile/data/CustomerTypes.kt
package com.gurucrm.mobile.data

object CustomerTypes {
    val all = listOf("retailer", "wholesaler", "supplier", "distributor", "dealer")

    fun label(type: String): String = when (type) {
        "retailer" -> "Retailer"
        "wholesaler" -> "Wholesaler"
        "supplier" -> "Supplier"
        "distributor" -> "Distributor"
        "dealer" -> "Dealer"
        else -> type.replaceFirstChar { it.uppercase() }
    }
}

File: crm-app/frontend/src/constants/customerTypes.js
export const CUSTOMER_TYPES = [
  { value: 'retailer', label: 'Retailer' },
  { value: 'wholesaler', label: 'Wholesaler' },
  { value: 'supplier', label: 'Supplier' },
  { value: 'distributor', label: 'Distributor' },
  { value: 'dealer', label: 'Dealer' },
];

export function customerTypeLabel(value) {
  return CUSTOMER_TYPES.find((t) => t.value === value)?.label || value;
}
